pack.yaml

```diff
@@ -6,7 +6,7 @@ keywords:
   - networking
   - ipam
   - dcim
-version: 2.0.0
+version: 2.1.0
```

Comment:
Considering such a massive diff, new actions, and the breaking changes, it would make more sense to bump the pack version to `v3.0.0`